books/list.html

```diff
@@ -9,6 +9,13 @@
         <link rel="stylesheet"  type="text/css" href="application.css">
     </head>
     <body>
+        <div class="book-navbar">
+            <div class="book-navbar-line"></div>
```

Comment:
I think this is not needed. Try to solve this using CSS 😉

Reply:
It was replaced by a border